### ISSUE TITLE
Prevent pushing typescripts's javascript output

### ISSUE THREAD

**Issue by guido-ferraro** [JuiceMind/JuiceMind#239]
`juicemind-cloud-functions` emitted JS shouldn't need to be pushed. This involves doing some research, could affect deployment process.

https://github.com/JuiceMind/juicemind-cloud-functions/tree/fd960ee1500bf4ff8891ba085bf695d963b84aff/functions/lib

**Comment by marklee45** [comment]
@guido-ferraro I'm going to unassign you from this is that ok?

**Comment by marklee45** [comment]
@guido-ferraro and what sev do you think this is?

**Comment by guido-ferraro** [comment]
5. The lowest

**Comment by guido-ferraro** [comment]
Yes, it's ok. A while ago I created like a million tickets of improvements like this one and I assigned all of them to myself because I didn't know, but Phil may be best suited as he handles the DevOps in general

**Comment by marklee45** [comment]
ya ok, got ya. I'm trying to clean up the issues right now.
